tests-asyncio/test_asyncio_based_service.py

```diff
@@ -217,6 +216,50 @@ async def run(self):
         await AsyncioManager.run_service(ServiceTest())
 
 
+@pytest.mark.asyncio
+async def test_daemon_task_cancels_service_leaving_children():
```

Comment:
I suppose this is the test that checks the `self.is_cancelled` path.

Reply:
Yep